pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarConnectorConfig.java

```diff
@@ -500,10 +513,19 @@ public PulsarAdmin getPulsarAdmin() throws PulsarClientException {
         if (this.pulsarAdmin == null) {
             PulsarAdminBuilder builder = PulsarAdmin.builder();
 
+            if (getAuthPlugin() != null && getAuthToken() != null) {
+                throw new IllegalArgumentException("Either pulsar.auth-plugin or pulsar.auth-token can be used,"
+                        + " but not both at the same time");
+            }
+
             if (getAuthPlugin() != null) {
                 builder.authentication(getAuthPlugin(), getAuthParams());
             }
 
+            if (getAuthToken() != null){
```

Comment:
```suggestion
            if (getAuthToken() != null) {
```

You may checkstyle locally with `mvn checkstyle:check`.